korebuild-lock.txt

```diff
@@ -1,2 +1,2 @@
-version:2.1.0-preview2-15742
-commithash:21fbb0f2c3fe4a9216e2d59632b98cfd7d685962
+version:2.1.0-preview3-15744

```

Comment:
Did this update when you did `build -update`? It shouldn't be picking up `preview3`.

Reply:
Could you re-run this?